From 28e911d61d66b92a3bded8b54622ed3cd2795bf6 Mon Sep 17 00:00:00 2001
From: Matt Bobrowski <matt@bobrowski.net>
Date: Sat, 1 Aug 2026 23:03:30 +1000
Subject: [PATCH 09722/15333] bpf: update BPF LSM maintainer list

I've recently left Google, so my mattbobrowski@google.com mail address
is now inactive. Update the BPF LSM maintainer entry with a mail
address that I do still have access to and use.

Note that I lost access to my mattbobrowski@google.com mail address
before being able to make this MAINTAINER entry change from it.

Signed-off-by: Matt Bobrowski <matt@bobrowski.net>
Link: https://lore.kernel.org/bpf/am3uorMW7_UWA5An@lima-development
Signed-off-by: Kumar Kartikeya Dwivedi <memxor@gmail.com>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 32ca23e58f2b..e46c57f8493c 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -5030,7 +5030,7 @@ F:	kernel/bpf/ringbuf.c
 
 BPF [SECURITY & LSM] (Security Audit and Enforcement using BPF)
 M:	KP Singh <kpsingh@kernel.org>
-M:	Matt Bobrowski <mattbobrowski@google.com>
+M:	Matt Bobrowski <matt@bobrowski.net>
 L:	bpf@vger.kernel.org
 S:	Maintained
 F:	Documentation/bpf/prog_lsm.rst
-- 
2.55.0

